frontends/mit-open/src/pages/HomePage/HomePage.test.tsx

```diff
@@ -117,7 +117,7 @@ describe("Home Page Carousel", () => {
       params: { resource_type: ["course"], limit: 12, certification: true },
     },
     {
-      tab: "Professional",
+      tab: "Professional and Executive Education",
```

Comment:
"Professional & Executive Learning"  , just because that's what it says here https://prolearn.mit.edu/